modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteNodeRestartTest.java

```diff
@@ -1832,8 +1850,10 @@ private Set<Assignment> getAssignmentsFromMetaStorage(MetaStorageManager metaSto
                 : Assignments.fromBytes(e.value()).nodes();
     }
 
-    private int tableId(Ignite node, String tableName) {
-        return (unwrapTableImpl(node.tables().table(tableName))).tableId();
+    private int zoneId(IgniteImpl node, String zoneName) {
+        int zoneId = getZoneIdStrict(node.catalogManager(), zoneName.toUpperCase(), node.clock().nowLong());
```

Comment:
Isn't better just return statement? `getZoneIdStrict` shows explicitly that the result is zoneId

Reply:
It is just a test. There are no far-reaching plans.
I don't think it needs to be improved.